kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/linux/util/LinuxProcessUtil.java

```diff
@@ -34,6 +34,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * @deprecated since 2.2 in favor of {@link org.eclipse.kura.executor.CommandExecutorService}
```

Comment:
I'm not sure if it is correct in the core bundle to mention the API version. In theory those packages are separate and not related.

Reply:
Indeed these are not "official" APIs, so they aren't related to the version...